include/oneapi/mkl/blas/detail/rocblas/blas_ct.hpp

```diff
@@ -0,0 +1,56 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Copyright (C) Heidelberg University, Engineering Mathematics and Computing Lab (EMCL) and Computing Centre (URZ)
```

Comment:
@mkrainiuk Could you please check if the Copyright banner is appropriate here and in other new documents?